server/app.js

```diff
@@ -43,6 +44,7 @@ app.use((req, res, next) => {
 
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
+app.use("/s3", s3Router);
```

Comment:
Would be clearer to have this route be something like `/upload`

Reply:
Sure, Will do that